### PR TITLE
Edit for grammar and style

### PR DESCRIPTION
**Pull request by john-andrilla**
In the part where you don't have enough context, I'd suggest not including exercises without proven benefits for the reader. You could always add more context or rationale in a narrative paragraph before the code sample.
@aaronmarkham Can you assist in supplying a less "mathy" demo of the control flows? Also, it's not clear just how the intro under Using Python control flows is connected to the code sample or to the subsection title. Can you add a transition to tie it together?

### CODE REVIEW

File: guide/crash-course/3-autograd.md [hunk 1]
```diff
@@ -1,58 +1,60 @@
...
 x.grad
 ```
 
 ## Using Python control flows
 
-Sometimes we want to write dynamic programs where the execution depends on some real-time values. MXNet will record the execution trace and compute the gradient as well.
+Sometimes you want to write dynamic programs where the execution depends on real-time values. MXNet records the execution trace and computes the gradient as well.
 
-Consider the following function `f`: it doubles the inputs until it's `norm` reaches 1000. Then it selects one element depending on the sum of its elements.
+Consider the following function `f`. The function doubles the inputs until it's `norm` reaches 1000. Then it selects one element depending on the sum of its elements. The following example code
```
Comment (aaronmarkham) [inline guide/crash-course/3-autograd.md]:
```suggestion
Consider the following function `f` in the following example code. The function doubles the inputs until its `norm` reaches 1000. Then it selects one element depending on the sum of its elements. 
```

File: guide/crash-course/3-autograd.md [hunk 2]
```diff
@@ -67,7 +69,7 @@ def f(a):
     return c
 ```
 
-We record the trace and feed in a random value:
+In this exmaple, you record the trace and feed in a random value.
```
Comment (aaronmarkham) [inline guide/crash-course/3-autograd.md]:
```suggestion
In this example, you record the trace and feed in a random value.
```

File: guide/crash-course/3-autograd.md [hunk 3]
```diff
@@ -77,12 +79,12 @@ with autograd.record():
 c.backward()
 ```
 
-We know that `b` is a linear function of `a`, and `c` is chosen from `b`. Then the gradient with respect to `a` be will be either `[c/a[0], 0]` or `[0, c/a[1]]`, depending on which element from `b` we picked. Let's find the results:
+You can see that `b` is a linear function of `a`, and `c` is chosen from `b`. The gradient with respect to `a` be will be either `[c/a[0], 0]` or `[0, c/a[1]]`, depending on which element from `b` is picked. You can find the results, with this code:
```
Comment (aaronmarkham) [inline guide/crash-course/3-autograd.md]:
```suggestion
You can see that `b` is a linear function of `a`, and `c` is chosen from `b`. The gradient with respect to `a` be will be either `[c/a[0], 0]` or `[0, c/a[1]]`, depending on which element from `b` is picked. You see the results of this example with this code:
```

**Review by aaronmarkham — CHANGES_REQUESTED** [review]
I don't know of a less mathy way to show the control flow for autograd.
A couple of typos are in here though.

**Comment by john-andrilla** [comment]
Thanks Aaron, committed all changes.

**Review by aaronmarkham — APPROVED** [review]

**Pull request merged**